spec/lib/facilities/gis_client_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+RSpec.describe Facilities::GisClient do
+  RSpec::Matchers.define :has_order_and_offset do |order, offset|
+    match do |params|
+      params[:orderByFields].eql?(order) && params[:resultOffset].eql?(offset)
+    end
+  end
+
+  context 'params' do
```

Comment:
This strikes me as a `describe` instead of a `context` and should maybe say `'get_all_facilities'`